examples/models/llava_encoder/install_requirements.sh

```diff
@@ -19,4 +21,7 @@ pip install bitsandbytes -I
 # For example, torch version required from llava is older than ExecuTorch.
 # To make both work, recover ExecuTorch's original dependencies by rerunning
 # the install_requirements.sh.
-./install_requirements.sh
+pip install --force-reinstall torch==2.3.0 --index-url https://download.pytorch.org/whl/test/cpu
+
+echo "Finished installing llava dependencies."
+pip list
```

Comment:
Could we have a `python -c 'import torch; print(torch.__version__)'` to confirm the torch version in used here?  I'm curious

Reply:
Maybe `python -c 'import torchvision; print(torchvision.__version__)'` and `python -c 'import torchaudio; print(torchaudio.__version__)'` too.  From what I try to reproduce locally, their versions looks correct:

```
(executorch) [huydo@devgpu018.odn1 ~/local/github/executorch (main)]$ python -c 'import torchvision; print(torchvision.__version__)'
0.19.0.dev20240324+cpu
(executorch) [huydo@devgpu018.odn1 ~/local/github/executorch (main)]$ python -c 'import torchaudio; print(torchaudio.__version__)'
2.2.0.dev20240324+cpu
```

Reply:
Added the debug log and removed `-I`. However, I don't even see `torch version = ` is printed hmm

Reply:
> I could see it in the log though, i.e. https://github.com/pytorch/executorch/actions/runs/8542428080/job/23404038664?pr=2818#step:11:6208

Okay, now I can see the version seems correct now and the job is going further, though it fails the failure is due to memory limits. I think Martin was asking about this in a thread and there is solution for it?